tensorboard/components/vz_chart_helpers/plottable-interactions.js

```diff
@@ -0,0 +1,144 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+var vz_chart_helpers;
+(function(vz_chart_helpers) {
```

Comment:
I assume this needs to be in JS in order to access visibility-protected members?  If that's the case a comment to the effect would be helpful.

Also some overall comment that explains why we're reimplementing Plottable.Interactions.Pointer would be useful.